packages/legacy/src/rest-3/paramsToString.native.ts

```diff
@@ -0,0 +1,10 @@
+export default function paramsToString(
+  searchParams: Readonly<Record<string, string | number>>,
+) {
+  const params = new URLSearchParams(searchParams as any);
+  try {
+    params.sort();
+    // eslint-disable-next-line no-empty
+  } catch {}
+  return params.toString();
```

Comment:
```suggestion
  return String(params ?? '');
```

Reply:
how does the existing fail? undefined?

Reply:
both undefined and null, yeah. the try/catch on the sort means you can't be sure it's coercible to an object by this point.

Reply:
I guess I'm confused how `new URLSearchParams(searchParams as any);` results in null or undefined

Reply:
oh, you're totally right. in this case it'd just be that if someone monkeyed with URLSearchParams.prototype.toString, the return might not actually be a string, whereas with `String(params)` it always will be.
```suggestion
  return String(params);
```
but i'll resolve this, it's not as important as i thought.